crowns.lic

```diff
@@ -117,27 +117,25 @@ class Dice
   end
 
   def check_prize
-    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
-      bput("put my #{righthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{righthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
-      end
-      pause
-    end
-    # Check left hand as well. On lag may have items in both hands.
-    if @trash_items.any? { |item| Regexp.new(item) =~ left_hand }
-      bput("put my #{lefthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{lefthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
+    [left_hand, right_hand]
+      .compact
+      .each do |in_hand|
+        if @trash_items.any? { |item| Regexp.new(item) =~ in_hand }
+          bput("put my #{in_hand} in bucket", 'You put', 'You drop')
+        else
+          # Coil rope so we can store it
+          if /\brope\b/ =~ in_hand
+            fput("coil my #{in_hand}")
+          end
```

Comment:
Coil rope otherwise too long to stow and script will abort prematurely

Reply:
I found this method was not always catching if the item was on the junk list or not. Rope is on the junk list. I went back to 
`thing = DRC.right_hand || DRC.left_hand
    waitrt?
    if thing
      case thing
      when *trash_items
        DRC.bput("put my #{thing} in bucket", 'You put', 'You drop')` and it catches everything better. 

I also consolidated the old lists into `hollow_eve_junk` for a new PR (have not yet put in). 

Could add `dice_loot_container:` at the same time.